pkg/migrations/sql/preinstall/001-users.sql

```diff
@@ -2,8 +2,11 @@ CALL SCHEMA_CATALOG.execute_everywhere('create_prom_reader', $ee$
     DO $$
         BEGIN
             CREATE ROLE prom_reader;
-        EXCEPTION WHEN duplicate_object THEN
-            RAISE NOTICE 'role prom_reader already exists, skipping create';
+        EXCEPTION
+            WHEN duplicate_object THEN
+                RAISE NOTICE 'role prom_reader already exists, skipping create';
+            WHEN others THEN
```

Comment:
I agree with Mat. If the `others` path executes, much of the rest of the preinstall script will likely fail since the role is required. I think we should ONLY catch the `duplicate_object` exception.